libs/common-hooks/src/useWidgetMode.ts

```diff
@@ -0,0 +1,16 @@
+import { isInjectedWidget } from '@cowprotocol/common-utils'
+import { WidgetThemeMode } from '@cowprotocol/types'
+
+export function useWidgetMode(): WidgetThemeMode {
+  const isWidget = isInjectedWidget()
+  const isIframe = window.self !== window.top
+  const isStandalone = !isWidget && !isIframe
+  const isInjectedWidgetMode = isWidget
+
+  return {
+    isInjectedWidgetMode,
+    isStandalone,
```

Comment:
`isIframe` and `isStandalone` seem to be unused.
So, we actually need only `isWidget` and `isIframe`.
SInce those values are not reactive (static), this hooks looks excessive.